domain/entity/Player.ts

```diff
@@ -1,10 +1,12 @@
 import { v4 as uuidv4 } from "uuid";
 import { Resources } from "./Resources";
+import { Tile } from "./Furnishing";
 
 export class Player {
     id: PlayerId = uuidv4();
     resources: Resources = new Resources();
     dwarfs: Map<DwarfId, Dwarf> = new Map();
+    tilesToPlace: Tile[] = [];
```

Comment:
I'm not sure that we should mutate the Player for this kind of thing, it looks a bit weird to me

But OK, we can decide that later, this can work for now

Reply:
Agreed, we probably should have a separate object for that